test/scrabble_test.rb

```diff
@@ -6,8 +6,27 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    game = Scrabble.new
```

Comment:
Would like to see some tests for lower case. Breaking the logic out (as mentioned above) would allow you to have some unit tests. Currently read as all integration.